bonfire/bonfire.py

```diff
@@ -1016,6 +1038,170 @@ def _cmd_apps_what_depends_on(
     print("\n".join(found) or f"no apps depending on {component} found")
 
 
+@reservation.command("create")
+@click.option(
+    '--requester',
+    '-r',
+    prompt='Requester',
```

Comment:
Can we default to using 'oc whoami' if user does not provide the requester

Reply:
Updated this to default to using 'oc whoami' whenever requester is not present. Note: 'oc whoami' will most likely error when running locally in minikube so in that case I set the requester to 'bonfire'